groupcache/src/groupcache_inner.rs

```diff
@@ -35,12 +37,23 @@ struct Config {
     grpc_endpoint_builder: Arc<Box<dyn Fn(Endpoint) -> Endpoint + Send + Sync + 'static>>,
 }
 
+async fn run_service_discovery<Value: ValueBounds>(
+    cache: Arc<GroupcacheInner<Value>>,
+    service_discovery: Arc<Box<dyn ServiceDiscovery + Send + Sync>>,
+) {
+    loop {
+        tokio::time::sleep(Duration::from_secs(10)).await;
+        let instances = service_discovery.instances().await.expect("");
```

Comment:
Error shouldn't crash service discovery loop.

Reply:
Sure thing, this is just for testing, probably made draft too soon